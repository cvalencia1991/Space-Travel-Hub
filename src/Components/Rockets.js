import { React } from 'react';

const Rockets = () => (
  <div>Rockets</div>
);

export default Rockets;
