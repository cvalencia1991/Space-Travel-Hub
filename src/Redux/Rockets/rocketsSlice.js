import { createSlice } from '@reduxjs/toolkit';

const initialState = [];
const rocketsSlice = createSlice({
  name: 'books',
  initialState,
});

export default rocketsSlice.reducer;
