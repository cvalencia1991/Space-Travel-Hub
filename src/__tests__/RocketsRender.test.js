import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Rockets from '../Components/Rockets';
import store from '../Redux/configureStore';

it('renders correctly', () => {
  const rockets = renderer
    .create(<Provider store={store}><Rockets /></Provider>)
    .toJSON();
  expect(rockets).toMatchSnapshot();
});
