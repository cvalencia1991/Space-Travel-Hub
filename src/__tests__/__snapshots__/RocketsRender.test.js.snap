// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="rocket-list m-5 p-3"
/>
`;
